### ISSUE TITLE
More build trouble in Webpack

### ISSUE THREAD

**Issue by jryans** [browserify/timers-browserify#34]
From @drcallaway:

It looks like this change is causing the following webpack build error:

```
/* WEBPACK VAR INJECTION */(function(global) {var scope = self || window;
                                                          ^

ReferenceError: self is not defined
    at Object.eval (webpack:///./node_modules/timers-browserify/main.js?:1:59)
```

**Comment by jryans** [comment]
@drcallaway, can you try editing your copy to use: 
```
var scope = (typeof self !== "undefined" && self) || window;
```

**Comment by drcallaway** [comment]
@jryans Yep. That fixed my problem.

**Comment by jryans** [comment]
Okay, I've published this fix in 2.0.9.
